stable/gocd/templates/configmap.yaml

```diff
@@ -51,11 +51,11 @@ data:
           },
           {
             "key": "PodConfiguration",
-            "value": "apiVersion: v1\nkind: Pod\nmetadata:\n  name: pod-name-prefix-{\\\\{ POD_POSTFIX }\\\\}\n  labels:\n    app: web\nspec:\n  containers:\n    - name: gocd-agent-container-{\\\\{ CONTAINER_POSTFIX }\\\\}\n      image: {\\\\{ GOCD_AGENT_IMAGE }\\\\}:{\\\\{ LATEST_VERSION }\\\\}\n      securityContext:\n        privileged: true"
+            "value": "apiVersion: v1\nkind: Pod\nmetadata:\n  name: pod-name-prefix-{{ `{{ POD_POSTFIX }}` }}\n  labels:\n    app: web\nspec:\n  serviceAccountName: {{ template "gocd.agentServiceAccountName" . }}\n  containers:\n    - name: gocd-agent-container-{{ `{{ CONTAINER_POSTFIX }}` }}\n      image: gocd/gocd-agent-docker-dind:v{{ .Chart.AppVersion }}\n      securityContext:\n        privileged: true"
```

Comment:
I haven't verified that this works as intended. As long as you have, should be fine. :)

Reply:
Yep, this works. The current version has invalid escaping and doesn't work.